richardaspinall/poker
# Add websocket connections and logic for showing and removing avatar

### Description

This PR adds the scaffolding for the websocket connection between the frontend and backend. Additionally to see if it works end to end I have added logic for showing and removing the avatar.

### How to test
1. Start the server (see readme)
2. Head to `localhost:3000` and click on a seat

Expected:
<img width="1072" alt="Screenshot 2023-04-16 at 5 09 56 pm" src="https://user-images.githubusercontent.com/15721687/232279319-29503ea8-c97a-4294-a062-c92ca7aa54ac.png">

### Task
https://app.clickup.com/t/860qhz7v8

### Checklist

- [ ] Learning objectives added to PR
